docs/pages/admin-guides/teleport-policy/crown-jewels.mdx

```diff
@@ -23,14 +23,13 @@ log in via Teleport Auth Connectors.
 
 - A running Teleport Enterprise cluster v16.2.0 or later.
 - For self-hosted clusters, an updated `license.pem` with Teleport Policy enabled.
-- For self-hosted clusters, a running Access Graph node v1.24.0 or later.
-Check [Access Graph page](teleport-policy.mdx) for details on
-how to set up Access Graph.
+- For self-hosted clusters, a running Graph Explorer node v1.24.0 or later.
```

Comment:
Technically we run access graph service for Teleport Policy https://gallery.ecr.aws/gravitational/access-graph